api/src/main/java/uk/gov/dhsc/htbhf/claimant/SchedulerConfig.java

```diff
@@ -0,0 +1,81 @@
+package uk.gov.dhsc.htbhf.claimant;
+
+import org.quartz.JobBuilder;
+import org.quartz.JobDetail;
+import org.quartz.SimpleScheduleBuilder;
+import org.quartz.Trigger;
+import org.quartz.TriggerBuilder;
+import org.quartz.spi.JobFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.scheduling.quartz.SchedulerFactoryBean;
+import uk.gov.dhsc.htbhf.claimant.scheduler.AutowiringSpringBeanJobFactory;
+import uk.gov.dhsc.htbhf.claimant.scheduler.CreateCardJob;
+
+import javax.sql.DataSource;
+
+/**
+ * Configuration for scheduling the creation of new cards.
+ */
+@Configuration
+public class SchedulerConfig {
+
+    /**
+     * Create a job factory that is spring context aware.
+     * @param applicationContext spring application context.
+     * @return spring aware job factory
+     */
+    @Bean
+    public JobFactory jobFactory(ApplicationContext applicationContext) {
+        AutowiringSpringBeanJobFactory jobFactory = new AutowiringSpringBeanJobFactory();
+        jobFactory.setApplicationContext(applicationContext);
+        return jobFactory;
+    }
+
+    /**
+     * Creates a job detail instance containing the job to create new cards.
+     * @return job detail
+     */
+    @Bean
+    public JobDetail createCardJobDetail() {
+        return JobBuilder.newJob(CreateCardJob.class)
+                .withIdentity("Create new cards")
+                .storeDurably()
+                .build();
+    }
+
+    /**
+     * Trigger the job every number of hours (value in configuration) and run indefinitely (first run is number of hours after start up).
+     * @param interviewInHours Number of hours to wait between every trigger. First run will be this many hours after startup.
+     * @return the job trigger
+     */
+    @Bean
+    public Trigger createCardTrigger(@Value("${card.new-card-interval-in-hours}") Integer interviewInHours) {
```

Comment:
I think this interval ought to be specifiable in minutes, not hours. We can have a default of 60, but lets have the flexibility to make it more frequent if needs be.

Reply:
Done.